teleop_twist_joy:
  enable_button: 1
  axis_linear: 1
  scale_linear: 0.6
  
  axis_angular: 0
  scale_angular: 0.8

  enable_turbo_button: 2
  scale_linear_turbo: 1.8
  scale_angular_turbo: 1.8
